buildenv/jenkins/JenkinsfileBase

```diff
@@ -451,14 +451,27 @@ def buildTest() {
 			if (fileExists('openjdkbinary/openjdk-test-image/openj9')) {
 				env.NATIVE_TEST_LIBS = "$WORKSPACE/openjdkbinary/openjdk-test-image/openj9"
 			}
-
-			// use sshagent with Jenkins credentials ID for all platforms except zOS
-			if (!env.SPEC.startsWith('zos')) {
-				sshagent (credentials: ["$params.SSH_AGENT_CREDENTIAL"], ignoreMissing: true) {
-					sh "./openjdk-tests/maketest.sh ./openjdk-tests";
+			
+			if(params.CUSTOMIZED_SDK_URL_CREDENTIAL_ID) {
+				withCredentials([usernamePassword(credentialsId: "${params.CUSTOMIZED_SDK_URL_CREDENTIAL_ID}", usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+					// use sshagent with Jenkins credentials ID for all platforms except zOS
+					if (!env.SPEC.startsWith('zos')) {
+						sshagent (credentials: ["$params.SSH_AGENT_CREDENTIAL"], ignoreMissing: true) {
+							sh "./openjdk-tests/maketest.sh ./openjdk-tests";
+						}
+					} else {
+						sh "./openjdk-tests/maketest.sh ./openjdk-tests";
+					}
 				}
 			} else {
-				sh "./openjdk-tests/maketest.sh ./openjdk-tests";
+				// use sshagent with Jenkins credentials ID for all platforms except zOS
+				if (!env.SPEC.startsWith('zos')) {
+					sshagent (credentials: ["$params.SSH_AGENT_CREDENTIAL"], ignoreMissing: true) {
+						sh "./openjdk-tests/maketest.sh ./openjdk-tests";
+					}
+				} else {
+					sh "./openjdk-tests/maketest.sh ./openjdk-tests";
+				}
```

Comment:
Instead of duplicating the code in the if-else block, could we make a function for this?

Reply:
Good idea! Updated the PR.